src/components/singleton-editor/useImageUploadSetup.ts

```diff
@@ -50,6 +53,11 @@ export function useImageUploadSetup({
           throw err;
         }
 
+        const etx = uploadFile.type.substring(6);
+        if (!IMAGE_FILE_ACCEPT.includes(etx)) {
+          throw new Error("file not accepted");
```

Comment:
No seria mas claro con la validacion hecha algo como: "File extension not accepted" o "File type not accepted"

Reply:
me gusta mas la primera